mkdocs.yml

```
@@ -61,6 +61,7 @@ nav:
   - getting-started.md
   - command-line-completion.md
   - CONTRIBUTING.md
+  - UPGRADING.md
```

```suggestion
  - UPGRADING.md
  - getting-started.md
  - command-line-completion.md
  - CONTRIBUTING.md
```
It seems better placed in the index directly under "Installation".